appinventor/blocklyeditor/src/msg/zh_cn/_messages.js

```diff
@@ -958,7 +958,7 @@ Blockly.Msg.zh.switch_language_to_chinese_cn = {
     Blockly.Msg.LANG_PROCEDURES_MUTATORCONTAINER_TITLE = '输入项';
     Blockly.Msg.LANG_PROCEDURES_MUTATORARG_TITLE = '输入:';
 
-    Blockly.Msg.LANG_PROCEDURES_HIGHLIGHT_DEF = '预览代码块功能';
```

Comment:
@farxinu What is this change doing here?

Reply:
This was because one of her changes originally built off of master, which had some additional changes to the translation. We should just be taking 46a8549, 6f26657, b89a950, 0a4eddf, and 8efe86f, squashed together. Ignore all the commits prior to 46a8549. I can also sit down with Farzeen tomorrow to rebase this change to make it clean.